src/logger.h

```diff
@@ -34,19 +34,24 @@ extern "C" {
 #include "transport.h"
 #include "loggerPrivate.h"
 
-///@cond INTERNAL
+/// @cond INTERNAL
 extern LoggerData g_logger;
-///@endcond
+/// @endcond
 
 /**
  * @brief Initialize the Logging sub-system.
  * @param tx_cb The callback to use for sending messages when
- *   Logging_SendResponse() is called.
- * @param enabled true to enable the logger, false to disable.
+ *   Logging_SendResponse() is called. This can be overridden, see the note
+ *   below.
  * @param max_payload_size The maximum size of the payload to be passed to the
  *   TXFunction. Must be at least 2.
+ *
+ * @note If PIPELINE_TRANSPORT_TX is defined in system_pipeline.h, the macro
+ * will override the tx_cb argument.
+ *
+ * A Logger starts off in the disabled state.
```

Comment:
Maybe , but I think we should be using less @note rather than more. 'If everything is important, then nothing is.'